### DISCUSSION TITLE
How to work together, how to contribute?

### DISCUSSION THREAD

**Discussion by annaesvensson** [datenstrom/community#760] [Ask a question] [answered]
Hello, here are some tips for working together.

**Comment by annaesvensson** [comment, marked as answer]
What you can do:

- Make a new feature - [start with an example feature](https://github.com/annaesvensson/yellow-example) and [make an extension](https://github.com/annaesvensson/yellow-publish)
- Make a new theme - [start with an example theme](https://github.com/annaesvensson/yellow-stockholm) and [make an extension](https://github.com/annaesvensson/yellow-publish)
- Make a new language - [start with an example language](https://raw.githubusercontent.com/annaesvensson/yellow-language/refs/heads/main/translations/english/english.php) and [make a translation](https://github.com/annaesvensson/yellow-language)

Where to find information for new developers:

- Get familiar with the [API for developers](https://datenstrom.se/yellow/help/api-for-developers), [customising HTML and CSS](https://datenstrom.se/yellow/help/how-to-customise-html-and-css)
- Dive deeper with the [core extension](https://github.com/annaesvensson/yellow-core), [edit extension](https://github.com/annaesvensson/yellow-edit) and [Markdown extension](https://github.com/annaesvensson/yellow-markdown)
- Become a contributor, [make an extension](https://github.com/annaesvensson/yellow-publish), [improve code and documentation](https://github.com/annaesvensson/yellow-help/blob/main/task-list.md)

Where to find information for new users:

- How to ask good questions? - [see #757](https://github.com/datenstrom/yellow-community-discussions/discussions/757)
- How to write good bug reports? - [see #759](https://github.com/datenstrom/yellow-community-discussions/discussions/759)
- Can I answer my own discussion? - [see #1032](https://github.com/datenstrom/yellow-community-discussions/discussions/1032)
- Why was my discussion flagged or deleted? - [see #734](https://github.com/datenstrom/yellow-community-discussions/discussions/734)

In summary, we offer a place to help each other and exchange knowledge. We organise our work around active people with talent, motivation and a genuine interest in working together. Datenstrom Yellow is open source. At the heart of open source is the idea that we make better software together.
